client/my-sites/plans/jetpack-plans/plans-filter-bar/index.tsx

```diff
@@ -113,37 +103,22 @@ const PlansFilterBar = ( {
 
 	return (
 		<div ref={ barRef } className={ classNames( 'plans-filter-bar', { sticky: hasCrossed } ) }>
-			{ showProductTypes && (
-				<SelectDropdown selectedText={ productType && PRODUCT_TYPE_OPTIONS[ productType ].label }>
-					{ Object.values( PRODUCT_TYPE_OPTIONS ).map( ( option ) => (
-						<SelectDropdown.Item
-							key={ option.id }
-							selected={ productType === option.id }
-							onClick={ () => onProductTypeChange?.( option.id ) }
-						>
-							{ option.label }
-						</SelectDropdown.Item>
-					) ) }
-				</SelectDropdown>
-			) }
-			{ showDurations && (
-				<SegmentedControl primary={ true }>
-					<SegmentedControl.Item
-						onClick={ () => onDurationChange?.( TERM_MONTHLY ) }
-						selected={ duration === TERM_MONTHLY }
-					>
-						{ translate( 'Monthly' ) }
-					</SegmentedControl.Item>
-
-					<SegmentedControl.Item
-						onClick={ () => onDurationChange?.( TERM_ANNUALLY ) }
-						selected={ duration === TERM_ANNUALLY }
-					>
-						{ translate( 'Yearly' ) }
-					</SegmentedControl.Item>
-				</SegmentedControl>
-			) }
-			{ showDiscountMessage && <DiscountMessage /> }
+			<SegmentedControl primary={ true }>
```

Comment:
`PlansFilterBar` is only used once now. We can safely remove the props that serve as conditionals.